test/adapter.js

```diff
@@ -27,7 +27,9 @@ function testFixture(description, fixture, generateSourceMap = false) {
       expected = fixture.apiElements;
     }
 
-    fury.parse({ source, generateSourceMap }, (err, output) => {
+    const mediaType = 'application/swagger+yaml';
```

Comment:
If we don't pass the media type for swagger to parse, the document won't be recognised as Swagger.